applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>Same points as before, I'd suggest something like:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># get data
filepath = DATASETS.fetch("CESM2_sst_data.nc")
ds = xr.open_dataset(filepath)

# calculate long term mean
tos_clim = ds.tos.groupby(ds.time.dt.month).mean(dim="time")

tos_clim
</pre><p> </p><p>And then create a subsection for the visualization</p><p> </p><p>(I'll stop making these at this point because they feel a bit redundant, assume that I'd also like to see matching changes in subsequent sections)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>